src/sentry/models/release.py

```diff
@@ -452,6 +454,7 @@ def set_commits(self, commit_list):
                     ],
                     last_commit_id=latest_commit.id if latest_commit else None,
                 )
+                metrics.timing('release.set_commits.duration', time() - start)
```

Comment:
Nbd, but we could use the `metrics.timer` context manager instead of calculating manually

Reply:
I didn't want to diff 300 lines for a 1 line change :blush:

Reply:
Fair :P